From a345204204898a3c54677382dc07f8855e49c070 Mon Sep 17 00:00:00 2001
From: Yi Zhao <yi.zhao@windriver.com>
Date: Fri, 11 Nov 2022 12:24:51 +0800
Subject: [PATCH] Fix sudo_remove_nopasswd

Upstream-Status: Inappropriate [WRLinux specific]

Signed-off-by: Yi Zhao <yi.zhao@windriver.com>
---
 shared/macros/bash.jinja | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/shared/macros/bash.jinja b/shared/macros/bash.jinja
index 1f4f9b143b..fa81ad831a 100644
--- a/shared/macros/bash.jinja
+++ b/shared/macros/bash.jinja
@@ -1345,7 +1345,7 @@ for f in /etc/sudoers /etc/sudoers.d/* ; do
   if ! test -z "$matching_list"; then
     while IFS= read -r entry; do
       # comment out "{{{ parameter }}}" matches to preserve user data
-      sed -i "s/^${entry}$/# &/g" $f
+      sed -i "s|^${entry}$|# &|g" $f
     done <<< "$matching_list"
 
     /usr/sbin/visudo -cf $f &> /dev/null || echo "Fail to validate $f with visudo"
-- 
2.25.1

